### PR TITLE
Fix Instance Segmentation Model

### PR DESCRIPTION
**Pull request by eugene123tw**
* Add `preload` arg to MaskRCNN and YOLACT
* Change `prob_threshold` to `confidence_threshold`

### CODE REVIEW

File: demos/common/python/openvino/model_zoo/model_api/models/instance_segmentation.py [hunk 1]
```diff
@@ -234,8 +234,8 @@ def postprocess(self, outputs, meta):
         return scores, classes, boxes, masks
 
     def _segm_postprocess(self, boxes, masks, score, classes, proto_data, w, h, shift_x=0, shift_y=0):
-        if self.prob_threshold > 0:
-            keep = score > self.prob_threshold
+        if self.confidence_threshold > 0:
+            keep = score > self.confidence_threshold
```
Comment (Wovchena) [inline demos/common/python/openvino/model_zoo/model_api/models/instance_segmentation.py:238]: What was wrong with `prob_threshold`? To me the variable name should follow demo flag name.

Comment (eugene123tw) [reply demos/common/python/openvino/model_zoo/model_api/models/instance_segmentation.py:238]: This is to keep consistency with SSD where `confidence_threshold` is used here (https://github.com/openvinotoolkit/open_model_zoo/blob/master/demos/common/python/openvino/model_zoo/model_api/models/ssd.py#L35). 

We are using this model with Model API for SC and would like to keep configuration consistency across models.

Comment (Wovchena) [reply demos/common/python/openvino/model_zoo/model_api/models/instance_segmentation.py:238]: This means we should change SSD to stay consistent with flags, not segmentation.

I though all integration with SC is executed by @akorobeinikov...

**Review by anzhella-pankratova — APPROVED** [review]

**Review by DmitriySidnev — APPROVED** [review]

**Review by akorobeinikov — APPROVED** [review]

File: demos/common/python/openvino/model_zoo/model_api/models/instance_segmentation.py (hunk 1 above)
Comment (Ilya-Krylov) [reply demos/common/python/openvino/model_zoo/model_api/models/instance_segmentation.py:238]: @Wovchena as far as I can see the most of models uses 'confidence_threshold'. Do you still insist to change it to 'prob_threshold'?


open_model_zoo/demos/common/python/openvino/model_zoo$ grep -r . -e confidence_threshold
```bash
./model_api/models/retinaface.py:        parameters['confidence_threshold'].update_default_value(0.5)
./model_api/models/retinaface.py:        outputs = self.postprocessor.process_output(outputs, scale_x, scale_y, self.confidence_threshold, self.mask_threshold)
./model_api/models/retinaface.py:        parameters['confidence_threshold'].update_default_value(0.5)
./model_api/models/retinaface.py:        outputs = self.postprocessor.process_output(outputs, scale_x, scale_y, self.confidence_threshold,
./model_api/models/yolo.py:        parameters['confidence_threshold'].update_default_value(0.5)
./model_api/models/yolo.py:    def _parse_yolo_region(cls, predictions, input_size, params, confidence_threshold):
./model_api/models/yolo.py:            keep_idxs = np.nonzero(class_probabilities > confidence_threshold)[0]
./model_api/models/yolo.py:            detections += self._parse_yolo_region(self, out_blob, meta['resized_shape'], layer_params[1], self.confidence_threshold)
./model_api/models/yolo.py:        parameters['confidence_threshold'].update_default_value(0.5)
./model_api/models/yolo.py:        valid_predictions = output[output[..., 4] > self.confidence_threshold]
./model_api/models/yolo.py:        i, j = (valid_predictions[:, 5:] > self.confidence_threshold).nonzero()
./model_api/models/yolo.py:        parameters['confidence_threshold'].update_default_value(0.5)
./model_api/models/yolo.py:        mask = np.array(out_scores) > self.confidence_threshold
./model_api/models/detection_model.py:            'confidence_threshold': NumericalValue(default_value=0.5, description="Threshold value for detection box confidence"),
./model_api/models/faceboxes.py:        mask = score > self.confidence_threshold
./model_api/models/detr.py:        parameters['confidence_threshold'].update_default_value(0.5)
./model_api/models/detr.py:        keep = det_scores > self.confidence_threshold
./model_api/models/centernet.py:        mask = detections[..., 4] >= self.confidence_threshold
./model_api/models/ultra_lightweight_face_detection.py:        parameters['confidence_threshold'].update_default_value(0.5)
./model_api/models/ultra_lightweight_face_detection.py:        mask = score > self.confidence_threshold
./model_api/models/ssd.py:        parameters['confidence_threshold'].update_default_value(0.5)
./model_api/models/ssd.py:        detections = [d for d in detections if d.score > self.confidence_threshold]
./model_api/models/ctpn.py:        parameters['confidence_threshold'].update_default_value(0.9)
./model_api/models/ctpn.py:        keep_inds = np.where((widths / heights > self.min_ratio) & (scores > self.confidence_threshold) &
```
open_model_zoo/demos/common/python/openvino/model_zoo$ grep -r . -e prob_threshold
```bash
./model_api/models/retinaface.py:    def process_output(self, raw_output, scale_x, scale_y, face_prob_threshold, mask_prob_threshold):
./model_api/models/retinaface.py:            threshold_mask = scores >= face_prob_threshold
./model_api/models/retinaface.py:                                                             0 if mask_scores_list[i] > mask_prob_threshold else 1,
./model_api/models/retinaface.py:    def process_output(self, raw_output, scale_x, scale_y, face_prob_threshold, image_size):
./model_api/models/retinaface.py:        filter_idx = np.where(scores > face_prob_threshold)[0]
./model_api/models/open_pose.py:        self.decoder = OpenPoseDecoder(num_joints, score_threshold=self.prob_threshold)
./model_api/models/open_pose.py:            'prob_threshold': NumericalValue(),
./model_api/models/instance_segmentation.py:            'prob_threshold': NumericalValue(
./model_api/models/instance_segmentation.py:        detections_filter = scores > self.prob_threshold
./model_api/models/instance_segmentation.py:            'prob_threshold': NumericalValue(
./model_api/models/instance_segmentation.py:            conf_mask = cls_scores > self.prob_threshold
./model_api/models/instance_segmentation.py:        if self.prob_threshold > 0:
./model_api/models/instance_segmentation.py:            keep = score > self.prob_threshold
./model_api/models/hpe_associative_embedding.py:            pose_threshold=self.prob_threshold,
./model_api/models/hpe_associative_embedding.py:            'prob_threshold': NumericalValue(),
```

3 approvals are not enough to merge?

Comment (Wovchena) [reply demos/common/python/openvino/model_zoo/model_api/models/instance_segmentation.py:238]: You argument with counting is strong) Thanks. We can keep `confidence_threshold` for now than.

**Pull request merged**